src/Components/ImagesTable/ImageLinkDirect.js

```diff
@@ -110,37 +110,44 @@ const ImageLinkDirect = ({ imageId, isExpired, isInClonesTable }) => {
   } else if (uploadStatus.type === 'gcp') {
     return (
       <Popover
-        aria-label="Popover with google cloud platform image details"
-        maxWidth="30rem"
-        headerContent={'GCP image details'}
+        aria-label="Popover with google cloud platform image commands"
+        maxWidth="29rem"
+        headerContent={'Image commands'}
         bodyContent={
           <TextContent>
-            <Text component={TextVariants.p}>
-              To use an Image Builder created Google Cloud Platform (GCP) image
-              in your project, specify the project ID and image name in your
-              templates and configurations.
-            </Text>
-            <Text>
-              <strong>Project ID</strong>
-              <br />
-              {uploadStatus.options.project_id}
-            </Text>
+            <br />
             <Text>
-              <strong>Image Name</strong>
-              <br />
-              {uploadStatus.options.image_name}
+              <strong>Launch an instance</strong>
             </Text>
+            <ClipboardCopy
+              hoverTip="Copy"
+              clickTip="Copied"
+              variant="inline-compact"
+              ouiaId="gcp-launch-instance"
+              isBlock
+            >
+              gcloud compute instances create INSTANCE_NAME --image-project
+              PROJECT_ID_FROM_RESPONSE --image IMAGE_NAME --zone GCP_ZONE
+            </ClipboardCopy>
+            <br />
             <Text>
-              <strong>Shared with</strong>
-              <br />
-              {/* the account the image is shared with is stored in the form type:account so this extracts the account */}
-              {image.uploadOptions.share_with_accounts[0].split(':')[1]}
+              <strong>Save a copy</strong>
             </Text>
+            <ClipboardCopy
+              hoverTip="Copy"
+              clickTip="Copied"
+              variant="inline-compact"
+              ouiaId="gcp-save-copy"
+              isBlock
+            >
+              gcloud compute images create MY_IMAGE_NAME --source-image-project
+              PROJECT_ID_FROM_RESPONSE --source-image IMAGE_NAME
```

Comment:
Wait I think the variables need to be filled in actually 
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/11140201/abb16058-4521-4e2f-8ff3-f524e5de1050)

Reply:
@croissanne Ok, I wasn't sure about this. Some of the variables are straightforward. If I get it right, the `INSTANCE_NAME` and `MY_IMAGE_NAME` would be identifiers set by user independently, right?

If that's correct we could do a partial fill. But what about `GCP_ZONE`? Is there a default we could use?

Reply:
It seems to set `ZONE` based on an environment variable if the tag is not defined, maybe? Can we just leave that part out?

Reply:
Indeed, but we can set the project_id, the IMAGE_NAME, and then we can prefill their name as "IMAGE_NAME-copy" maybe?


For the instance maybe

INSTANCE_NAME=IMAGE_NAME-instance ; GCP_ZONE we can leave as blank I guess.

Reply:
Sounds good, I've pushed the changes!